src/spatial/math.js

```diff
@@ -688,7 +697,7 @@ Crafty.math.Vector2D = (function () {
      *
```

Comment:
The previous method `Vector2D.prototype.subtract` has invalid `@sign` and `@param`